### PR TITLE
docs: update documentation for 0.7

### PR DESCRIPTION
**Pull request by rafaeldtinoco**

### CODE REVIEW

**Review by danielpacak — CHANGES_REQUESTED** [review]
The updates are looking great. Still reviewing the whole PR, but I'm sharing my feedback regarding the v prefix in image tags that we added in v0.7.0-rc-1. This has impact on places where we use `{{ git.tag[1:] }}` macros. IMO we'd rather publish 0.7.0 instead of v0.7.0 to make documentation updates easier.

File: docs/index.md [hunk 1]
```diff
@@ -7,60 +7,101 @@
...
+run:
 
-!!! note
-    You may need to change the volume mounts for the kernel headers based on your setup. See [Linux Headers](./install/headers.md) section for more info.
+```strace ls```
 
-This will run Tracee with default settings and start reporting detections to standard output.
-In order to simulate a suspicious behavior, you can run `strace ls` in another terminal, which will trigger the "Anti-Debugging" signature, which is loaded by default.
+in another terminal. This will trigger the "Anti-Debugging" signature, which is
+loaded by default AND you will get a warning.
```
[the remark this answers is not in the document]
Comment (grantseltzer) [reply docs/index.md:68]: So dramatic @rafaeldtinoco with the capital AND

**Comment by rafaeldtinoco** [comment]
@grantseltzer and @danielpacak since I'm traveling tomorrow morning (not sure about inet access) and we're releasing tomorrow, mind taking over this PR (even if you close this one and merge another one) so you both can finish the review/adjustments ?

File: docs/index.md [hunk 2]
```diff
@@ -7,60 +7,101 @@
...
-If running on __BTF enabled kernel__:
+1. Running **tracee:latest**
+
+   ```bash
+   $ docker run \
+        --name tracee --rm -it \
+        --pid=host --cgroupns=host --privileged \
+        -v /etc/os-release:/etc/os-release-host:ro \
+        -e LIBBPFGO_OSRELEASE_FILE=/etc/os-release-host \
+        aquasec/tracee:{{ git.tag[1:] }}
```
[the remark this answers is not in the document]
Comment (danielpacak) [reply docs/index.md:33]: Resolving this discussion as we've merged #1586

File: docs/index.md [hunk 3]
```diff
@@ -7,60 +7,102 @@
...
-Tracee is a Runtime Security and forensics tool for Linux. It is using Linux eBPF technology to trace your system and applications at runtime, and analyze collected events to detect suspicious behavioral patterns. It is delivered as a Docker image that monitors the OS and detects suspicious behavior based on a predefined set of behavioral patterns.
+Tracee is a Runtime Security and forensics tool for Linux. It is using Linux
+eBPF technology to trace your system and applications at runtime, and analyze
+collected events to detect suspicious behavioral patterns. It is delivered as a
+Docker image that monitors the OS and detects suspicious behavior based on a
+predefined set of behavioral patterns.
 
 Watch a quick video demo of Tracee: <br>
-<a href="https://youtu.be/9qxaOYto_5g"><img src="http://i3.ytimg.com/vi/9qxaOYto_5g/maxresdefault.jpg" width="400"></a>
+<a href="https://youtu.be/x2_iF0KjPKs?t=2971"><img src="https://i2.paste.pics/b755d5ee03048e3782f42da9870630eb.png?trs=a87585fdab9f70820cff773222a23ad3bbbc31d0b579bdc1b0ad91aa0cc19ecf" width="400"></a>
```
[the remark this answers is not in the document]
Comment (danielpacak) [reply docs/index.md]: I've added this image to docs/images/tracee_video_thumbnail.png

File: docs/index.md [hunk 4]
```diff
@@ -7,60 +7,101 @@
...
 
 ## Quickstart
 
 Before you proceed, make sure you follow the [minimum requirements for running Tracee](install/prerequisites.md).
 
-If running on __BTF enabled kernel__:
+1. Running **tracee:latest**
+
+   ```bash
+   $ docker run \
```
[the remark this answers is not in the document]
Comment (danielpacak) [reply docs/index.md]: I've done this actually.

File: docs/index.md [hunk 5]
```diff
@@ -7,60 +7,101 @@
...
+        --name tracee --rm -it \
+        --pid=host --cgroupns=host --privileged \
+        -v /etc/os-release:/etc/os-release-host:ro \
+        -e LIBBPFGO_OSRELEASE_FILE=/etc/os-release-host \
+        aquasec/tracee:{{ git.tag[1:] }}
+   ```
+
```
[the remark this answers is not in the document]
Comment (danielpacak) [reply docs/index.md]: Fixed Markdown so it's rendered as numbered list.

File: docs/index.md [hunk 6]
```diff
@@ -7,60 +7,101 @@
...
+!!! note
+    See documentation or add the `--help` flag for more.
 
 ## Components
 
-Tracee is composed of the following sub-projects, which are hosted in the aquasecurity/tracee repository:
+Tracee is composed of the following sub-projects, which are hosted in the
+aquasecurity/tracee repository:
 
 - [Tracee-eBPF](https://github.com/aquasecurity/tracee/tree/{{ git.tag }}/tracee-ebpf) - Linux Tracing and Forensics using eBPF
```
[the remark this answers is not in the document]
Comment (danielpacak) [reply docs/index.md]: I've updated these two links as well.

File: docs/index.md [hunk 7]
```diff
@@ -7,60 +7,101 @@
...
 ## Trace
 
-In some cases, you might want to leverage Tracee's eBPF event collection capabilities directly, without involving the detection engine. This might be useful for debugging/troubleshooting/analysis/research/education. In this case you can run Tracee with the `trace` sub-command, which will start dumping raw data directly into standard output. There are many configurations and options available so you can control exactly what is being collected and how. see the Documentation or add the `--help` flag for more.
+In some cases, you might want to leverage Tracee's eBPF event collection
+capabilities directly, without involving the detection engine. This might be
+useful for debugging/troubleshooting/analysis/research/education. In this case,
+you can run tracee exporting `TRACEE_EBPF_ONLY=1` environment variable.
+
+```bash
+$ docker run \
```
[the remark this answers is not in the document]
Comment (danielpacak) [reply docs/index.md]: Done for the docs/index.md page.

**Comment by danielpacak** [comment]
> @grantseltzer and @danielpacak since I'm traveling tomorrow morning (not sure about inet access) and we're releasing tomorrow, mind taking over this PR (even if you close this one and merge another one) so you both can finish the review/adjustments ?

Let me take this over @rafaeldtinoco and push some updates to your branch. And I'll let @grantseltzer review and merge it if everything looks good.

**Review by danielpacak — APPROVED** [review]

**Pull request merged**